src/common/websocket/config.js

```diff
@@ -19,6 +19,10 @@ export const domain_config = {
             hostname: 'deriv.me',
             app_id: 1411,
         },
+        {
+            hostname: 'deriv.be',
+            app_id: 30767,
+        },
```

Comment:
Can we update this and the above to use the variables from `constants.js`? Also noticed that the one for deriv.com is different with the one in constants.js. Could you help to check on that @khalid-deriv?

Reply:
Done!
And for deriv.com the second app ID also does the same thing, so I removed one for consistency (after checking with @habib-deriv)